e2e/tests/nightly/egress-suspension.spec.ts

```diff
@@ -36,17 +41,26 @@ describe('egress suspension', () => {
     await treePage.waitForTimeout(5000);
     for (let i = 0; i < 6; i++) {
       const f = `data${i}.txt`;
-      console.log(`Downloading ${f} to generate egress`);
+      logger.info(`Downloading ${f} to generate egress`);
       await notebookPage.downloadFileFromTree(treePage, f);
     }
+    await treePage.close();
+
+    logger.info('Awaiting security suspension in the notebook page');
+    await page.bringToFront();
+    notebookUrl = page.url();
+    await notebookPage.waitForLoad();
 
-    console.log('Awaiting security suspension in a new page');
-    const newPage = await browser.newPage();
-    await signInWithAccessToken(newPage, config.EGRESS_TEST_USER);
-    // Open the current notebook URL again; this will instead show a suspended
-    // error on reload once the user becomes suspended.
-    await newPage.goto(page.url());
+    await waitForSecuritySuspendedStatus(page);
+  });
+
+  test('Cloud Analysis Environment is auto suspended', async () => {
```

Comment:
What is the purpose of this test case beyond what the original test is doing? In general I'd prefer not to have two interdependent tests like this

Reply:
The lock status check is fine, but maybe just add this to the end of the other test. I don't think it's worth the complexity of a second dependent test case for the refresh case you describe. The suspension window is not that long (10m), so I really don't want tests that depend on the user being in this state for a prolonged time.

Reply:
Got it. It's done. Updated PR description with a new screenshot as well. Thank you.